.github/workflows/title_validation.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+filtered=$(echo $1 | egrep '(feat|fix|chore|build|docs|doc|refactor|techdebt|release|test|chore|BREAKING CHANGE)(\((web|agent|ci|cli|deps|crux|crux-ui|kratos|crane|dagent|test|doc)\))?: [\w\S]*')
```

Comment:
As per https://www.phoronix.com/news/GNU-Grep-3.8-Stop-egrep-fgrep  instead of egrep grep -E is to be used.

Reply:
Fixed.